README.md

```diff
@@ -146,8 +147,7 @@ This leads to two alternative ways of managing a new LevelUP instance:
 levelup(location, options, function (err, db) {
   if (err) throw err
   db.get('foo', function (err, value) {
-    if (err) return console.log('foo does not exist')
-    console.log('got foo =', value)
+    if (!err) console.log('got foo =', value)
```

Comment:
Want me to revert this change or maybe make it more complete, just like the first example?